schemas/top-api.yml

```diff
@@ -386,112 +371,112 @@ paths:
                 $ref: '#/components/schemas/PipelineResponse'
         '401':
           $ref: '#/components/responses/Unauthorized'
-        '404':
-          $ref: '#/components/responses/NotFound'
         default:
           $ref: '#/components/responses/UnexpectedError'
       tags:
-        - conceptgraphs
-    post:
-      operationId: startConceptGraphPipeline
-      summary: 'Initializes a complete pipeline with all steps ''preprocessing'', ''embedding'', ''clustering'' & ''graph creation''.'
+        - conceptPipeline
+    put:
+      operationId: startConceptGraphPipelineWithJson
```

Comment:
I'm not really sure/pleased about/with this, but there is no way (as far as I found) to have multiple `content`s in a `requestBody` to make this work in another way.

Reply:
Do you want "multipart/form-data" as content type?

Reply:
I can apply a commit to show you what I mean.

Reply:
See [5ad88ba](https://github.com/Onto-Med/top-api/pull/175/commits/5ad88ba889630e563e2373cb3b02061c73950ae7)

Reply:
Hm, when I did it like that, the openapi generator warned me, that there were two contents in the requestBody and only the first was taken...

Reply:
This is valid according to the OpenAPI specification, but it could be the case that the generator does not support multiple content types, which would be strange. I'll check that.

Reply:
Ok it is not working... I will revert the commit, hopefully this will be fixed in a new version of the generator. (see https://github.com/OpenAPITools/openapi-generator/issues/3990)